packages/backend/src/build-disk-image.ts

```diff
@@ -304,8 +307,24 @@ export async function getUnusedName(name: string): Promise<string> {
   return unusedName;
 }
 
+export async function getBuilder(): Promise<string> {
```

Comment:
Hmm the commit message mentions the `bootc.diskimage-builder` label but I don't see it mentioned or used in the code anywhere?

Oh except...argh, we may have lost that label in some reworking of the base image builds.

Reply:
We didn't see it used consistently enough in the images (and sounds like it isn't going to be?), so the extension still uses a simple builder preference (under Settings > Preferences > Bootable Containers in the UI).